sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -416,6 +416,36 @@ def list_containers(
                 page_iterator_class=ContainerPropertiesPaged
             )
 
+    @distributed_trace
+    def filter_blobs(self, where=None, **kwargs):
```

Comment:
hmm let me think about this name....
We're calling it this across all languages?

Reply:
It is a convention to start APIs returning ItemPaged to start with list_  - i know list_blobs exists - but still a little unconvinced with the name.

Reply:
how about find_blobs_by_tags? Java is using this name

Reply:
What does it mean if `where=None`? Is that the same behaivour as "list all blobs in all containers"?

Reply:
renamed to filter_expression